### PR TITLE
Update metrics processing notebook

### LINKED ISSUE #18: Automate metric collection and processing workflow
For creating a self updating metrics dashboard, we will have to create a pipeline that runs everyday and updates the data reflected in the dashboard. 

This pipeline will have following steps:

1. Data collection from Gtihub and storage on ceph
2. Metric calculation from raw Github data
3. Update results csv on Ceph being used by Trino and Superset

### LINKED ISSUE #23: Add sigs information to data tables
For grouping performance metrics based on special interest groups (SIGs), we have to add a column to the existing data tables with sig information it belongs to. The sig information should be parsed automatically from github.

Acceptance

- [x] Dataset for Superset graphs have a sigs column

### PR DESCRIPTION
**Pull request by chauhankaranraj**
In this PR, we
* parse sigs, orgs, repos from sigs.yaml instead of passing as them as an individual set of env variables one by one
* wrap metrics processing in functions that can be called in a for loop - this way, when the cronjob is run, it will collect data and update tables for _all_ repos in sigs.yaml instead of just one

Closes #18 
Closes #23

### CODE REVIEW

File: notebooks/Repo_Analysis.ipynb [hunk 1]
```diff
@@ -33,6 +33,8 @@
    ],
```
Comment (MichaelClifford) [inline notebooks/Repo_Analysis.ipynb:33]: <pre class="ql-syntax" spellcheck="false">Line #15.                    !python -m srcopsmetrics.cli -clr $org/$repo -e Issue,PullRequest # noqa: E999
</pre><p>Is there a more python friendly way of doing this? can srcopsmetrics be imported ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (MichaelClifford) [inline notebooks/Repo_Analysis.ipynb:33]: <p>Do we want to do this every time? or only an an initial run?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (chauhankaranraj) [reply notebooks/Repo_Analysis.ipynb:33]: Yep, we wanna create tables just on the initial run. This func will get called every time but I believe it won't create tables again coz it uses `CREATE TABLE IF NOT EXISTS`

**Pull request merged**